py-isort.el

```diff
@@ -36,6 +37,12 @@
   :type '(repeat (string :tag "option")))
 
 
+(defcustom py-isort-executable "isort"
+  "Name or path to the isory executable"
+  :group 'py-isort
+  :type '(repeat (string :tag "option")))
```

Comment:
This should just be type `string` and not a list of strings.

But unfortunately I also think this project is dead as @paetzke  doesn't seem to be active on GitHub anymore. (E.g. see https://github.com/paetzke/py-isort.el/pull/13 )